post_office/test_settings.py

```diff
@@ -37,6 +37,8 @@
         'connection_tester': 'post_office.tests.test_mail.ConnectionTestingBackend',
     },
     'MAX_RETRIES': 2,
+    'MESSAGE_ID_ON_DELIVERY': False,
+    'MESSAGE_ID_FQDN': 'post_office.org',
```

Comment:
We should rename this to `example.org` in case people copy this by mistake.

Reply:
OK